### PR TITLE
[AC-6851] People Directory - Use people profile IDs not Team member IDs

### PR DESCRIPTION
**Pull request by kitaka**
**Changes Introduced in [AC-6851](https://masschallenge.atlassian.net/browse/AC-6851):**
-

**Test:**

_On development or Staging_
- Visit people directory http://localhost:1234/people or https://staging.masschallenge.org/people
- Search for `des pieri`
- Notice that clicking the user, the people directory just reloads

_checkout this branch or Test5_
- Visit people directory http://localhost:1234/people or https://test5.masschallenge.org/people
- Search for `des pieri`
- Notice that this time around, clicking the user shows the profile

Sibling PR: https://github.com/masschallenge/front-end/pull/171

### CODE REVIEW

**Comment by mkiterian** [comment]
Taking code

**Review by kaymation — APPROVED** [review]
Tried to break the directory using the search bar but i couldnt! approved

**Comment by mkiterian** [comment]
@kitaka what's special about `des pieri`?

File: web/impact/impact/tests/test_graphql.py [hunk 1]
```diff
@@ -336,7 +333,7 @@ def test_query_with_non_existent_startup_team_member_id(self):
                 error_messages = [x['message'] for x in
                                   response.json()['errors']]
 
-            self.assertIn(TEAM_MEMBER_NOT_FOUND_MESSAGE, error_messages)
+            self.assertIn(ENTREPRENEUR_NOT_FOUND_MESSAGE, error_messages)
```
Comment (mkiterian) [inline web/impact/impact/tests/test_graphql.py:336]: the name of this test should also be updated

**Review by mkiterian — CHANGES_REQUESTED** [review]
Code changes look good. Minor cleanup requested

**Comment by kitaka** [comment]
> @kitaka what's special about `des pieri`?

Des Pieri is an expert; and since the original implementation only linked to profiles of startup team members, he's profile wasn't available. For that reason, he provided a good test case.

**Review by mkiterian — APPROVED** [review]
Looks good

**Pull request merged**